packages/amplify-graphql-sql-transformer/src/graphql-sql-transformer.ts

```diff
@@ -186,7 +182,7 @@ const getStatement = (config: SqlDirectiveConfiguration, customQueries: Map<stri
     );
   }
 
-  const statement = config.statement ?? customQueries.get(config.reference!);
+  const statement = config.statement ?? getCustomQuery(config.reference!);
   return statement!;
```

Comment:
Don't really like the typing here. At the least we should throw an error if statement is not found.

Reply:
It's safe to do--looking at the code above, you can see that the conditions are accurately checked--but I am not a fan of the force-unwraps (sorry, my Swift is showing) either. Will adjust.

Reply:
Fixed in 28a518056